src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -276,7 +275,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Runtime\InteropServices\WindowsRuntime\CLRIKeyValuePairImpl.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\System\AggregateException.cs" />
```

Comment:
The name of the serializable `m_innerExceptions` field is different between CoreCLR and CoreRT. It should have been detected by the binary serialization tests. Do we have a test hole in corefx for this?